src/main/java/moze_intel/projecte/gameObjs/gui/GUICollectorMK1.java

```diff
@@ -19,7 +19,7 @@ public GUICollectorMK1(InventoryPlayer invPlayer, CollectorMK1Tile tile)
 		super(new CollectorMK1Container(invPlayer, tile));
 		this.tile = tile;
 	}
-	
+
```

Comment:
idk why it's doing this, I think probably a tab was there and IDEA made it a real newline